tests/model/test_deep_speech_1.py

```diff
@@ -36,6 +41,46 @@ def deep_speech_1s(
 # Tests -----------------------------------------------------------------------
 
 
+@given(deep_speech_1s(return_kwargs=True))
+def test_correct_ds1_returned(ds1_kwargs: Tuple[DeepSpeech1, Dict]) -> None:
+    """Ensures correct ``rnn`` type and initialisation."""
+    ds1, kwargs = ds1_kwargs
+    fc1 = ds1.fc1
+    fc2 = ds1.fc2
+    fc3 = ds1.fc3
+    fc4 = ds1.fc4
+    fcout = ds1.out
+
+    if not isinstance(fc1, torch.nn.Linear):
+        fc1 = fc1[0]
```

Comment:
What are these `isinstance` checks for? Maybe worth commenting?

Reply:
These are to extract the fc layer if it is in a `Sequential` with dropout/activation (i.e due to the following line in the way the ds1 fully_connected are initialized: https://github.com/MyrtleSoftware/myrtlespeech/blob/d1cc3a8f35798e4df9f78f94d033464b80dbd6c4/src/myrtlespeech/model/deep_speech_1.py#L122